src/server/editorServices.ts

```diff
@@ -502,7 +522,11 @@ namespace ts.server {
             return scriptInfo && scriptInfo.getDefaultProject();
         }
 
-        private ensureInferredProjectsUpToDate() {
+        /**
+         * Ensures the project structures are upto date

```

Comment:
Will rename refreshInferredProjects  to forceRefreshInferredProjects. The difference being that even if the project graphs for the changed files/project update graphs pending do not update any structure, you want to ensure that inferred projects are upto date no matter what.